templates/verify.txt

```diff
@@ -1,6 +1,6 @@
-{{t "Almost there"}}
+{{t "Welcome!"}}
```

Comment:
@ryanfeeley did we discover that exclamation marks tend to be a bit of a spam trigger?  I have vague recollections of removing them long ago, but it might have had more to do with the whole phrase "Congratulations!" than with just exclamation marks in general.